topology/src/main/java/io/camunda/zeebe/topology/api/TopologyManagementRequestsHandler.java

```diff
@@ -83,4 +115,74 @@ private ActorFuture<TopologyChangeStatus> applyOperations(
 
     return responseFuture;
   }
+
+  private List<TopologyChangeOperation> generatePartitionDistributionOperations(
```

Comment:
:thought_balloon: Kinda feels like something that belongs in a different/new class.